### PR TITLE
Added theme to events documentation

### LINKED ISSUE #5902: "Events Documentation. " link in Developer Tools Events not themed
## Checklist

- [X ] I have updated to the latest available Home Assistant version.
- [X ] I have cleared the cache of my browser.
- [X ] I have tried a different browser to see if it is related to my browser.

## The problem

On the Developer Tools / Events, the link to the documentation it's not using the theme colors.

I checked with different themes and the issue is always there.

## Expected behavior

The color of the "Events Documentation. " link should look like all the links in the /Available Events/ list on the same page. 

image

## Steps to reproduce

Visit the page Developer Tools, on the Events tab.

## Environment

Frontend version: 20200427.2 - latest

- Home Assistant release with the issue:
Home Assistant 0.109.6
- Last working Home Assistant release (if known):
  N/A
- Browser and browser version:
  Chrome / Firefox.. latest versions.
- Operating system:
  arch | x86_64  
  -- | -- 
  dev | false 
  docker | true
  hassio | true
  os_name | Linux
  os_version | 4.19.115
  python_version | 3.7.7
  timezone | America/Curacao
  version | 0.109.6
  virtualenv | false

## State of relevant entities

```yaml

```

## Problem-relevant configuration

```yaml

```

## Javascript errors shown in your browser console/inspector

```txt

```

### PR DESCRIPTION
**Pull request by matstrange**
## Proposed change

Fixes bug #5902 - adds theme to urls to fix inconsistency as shown for external event documentation. Test screenshot below:

## Type of change

- [ ] Dependency upgrade
- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #5902
- This PR is related to issue:
- Link to documentation pull request:

## Checklist

- [x] The code change is tested and works locally.
- [x] There is no commented out code in this PR.
- [x] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/panels/developer-tools/event/developer-tools-event.js [hunk 1]
```diff
@@ -51,6 +51,10 @@ class HaPanelDevEvent extends EventsMixin(LocalizeMixin(PolymerElement)) {
           max-width: 800px;
           margin: 16px auto;
         }
+
+        a {
+          color: var(--dark-primary-color);
```
Comment (bramkragten) [inline src/panels/developer-tools/event/developer-tools-event.js:56]: We use primary color for links

**Review by bramkragten — CHANGES_REQUESTED** [review]

File: src/panels/developer-tools/event/developer-tools-event.js (hunk 1 above)
Comment (matstrange) [reply src/panels/developer-tools/event/developer-tools-event.js]: Updated (and on events-list which was also using dark

**Comment by matstrange** [comment]
image

 Updated test screenshot

**Review by zsarnett — APPROVED** [review]

**Comment by matstrange** [comment]
Found a couple more link styles in dev tools that were also using dark-primary so have fixed those too

**Review by bramkragten — APPROVED** [review]

**Pull request merged**